src/sdk/clients/decorators/erc7579/erc7579.decorators.test.ts

```diff
@@ -31,13 +39,13 @@ describe("erc7579.decorators", async () => {
 
     chain = network.chain
     bundlerUrl = network.bundlerUrl
-    account = getTestAccount(0)
+    eoaAccount = getTestAccount(0)
```

Comment:
Can you change this everywhere if you wanna change it? account seemed fine for me but no biggie